Define HAVE_UNISTD_H in config.h for the benefit of slrip/debug.c,
which ends up calling close without including <unistd.h> without the
macro definition.  This avoids build failures with future compilers
due to implicit function declarations.

Upstream no longer has this issue, presumably after the switch to
CMake.

diff --git a/configure.ac b/configure.ac
index 407b05785bbf081c..8e6f41cbe392ce33 100644
--- a/configure.ac
+++ b/configure.ac
@@ -10,7 +10,7 @@ AM_INIT_AUTOMAKE
 AC_PROG_CC
 AC_PROG_CXX
 AM_PROG_CC_C_O
-
+AC_CHECK_HEADERS([unistd.h])
 
 
 AC_MSG_CHECKING([whether to build for release])
